packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -138,11 +139,20 @@ export const startPSM = async (
     }),
   );
 
-  psmInstanceR.resolve(await E(governorFacets.creatorFacet).getInstance());
-  psmGovernorR.resolve(governorFacets.instance);
-  psmCreatorFacet.resolve(E(governorFacets.creatorFacet).getCreatorFacet());
-  psmAdminFacet.resolve(E(governorFacets.creatorFacet).getAdminFacet());
-  psmGovernorCreatorFacet.resolve(governorFacets.creatorFacet);
+  /** @typedef {import('./econ-behaviors.js').PSMFacets} PSMFacets */
+  /** @type {PSMFacets} */
+  const newPsmFacets = {
+    psm: E(governorFacets.creatorFacet).getInstance(),
+    psmGovernor: governorFacets.instance,
+    psmCreatorFacet: E(governorFacets.creatorFacet).getCreatorFacet(),
+    psmAdminFacet: E(governorFacets.creatorFacet).getAdminFacet(),
+    psmGovernorCreatorFacet: governorFacets.creatorFacet,
+  };
```

Comment:
I wonder if we're going to regret storing these promises.

Reply:
You're probably right.